libs/libbat/include/verge/mission-command/BatteryModel.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+#include <string>
+#include <verge/mission-command/BatteryCoefficients.h>
+#include <math.h>
```

Comment:
move to .cpp file (it's not needed here). Also, use `#include <cmath>` instead of `#include <math.h>`. both are valid, but the former is for c++